examples/static-site/build/about/index.html

```diff
@@ -0,0 +1,49 @@
+<!doctype html>
```

Comment:
Funny, I actually[ said the opposite a couple weeks ago](https://github.com/metalsmith/metalsmith/pull/237#discussion_r66503096) :wink: :

> Since these are meant to be used as examples, it's helpful for users to be able to see the expected result of the build right here on GitHub without having to check out the code and run it themselves.

Thoughts?